element/index.js

```diff
@@ -2,7 +2,7 @@
  * External dependencies
  */
 import { createElement, Component, cloneElement, Children } from 'react';
-import { render, findDOMNode, unstable_createPortal } from 'react-dom'; // eslint-disable-line camelcase
+import { render, findDOMNode } from 'react-dom'; // eslint-disable-line camelcase
 import { renderToStaticMarkup } from 'react-dom/server';
```

Comment:
We should replace this with `preact-render-to-string`

https://github.com/developit/preact-render-to-string